packages/ui-bindings/EconomicEvent/EconomicEventById.tsx

```diff
@@ -0,0 +1,49 @@
+import { connect } from "react-redux"
+import { gql, graphql, compose } from "react-apollo"
+import { AppState } from "@vflows/store/types"
+import { getActiveLoginToken } from "@vflows/store/selectors/auth"
+import { create } from "domain";
+
+export const query = gql`
+query($token: String, $id: Int) {
+  viewer(token: $token) {
+    economicEvent(id: $id) {
+      id
```

Comment:
See comments above on fragments and completeness

Reply:
Fragment created
:+1: